### PR TITLE
PIM-6873: add import/export informations about product model & family variant

### PR DESCRIPTION
**Pull request by Doodoune**
Update tech documentation for products with variants https://docs.akeneo.com/latest/import_and_export_data/formats.html

* Add parent in product format
* Add product model format
* Add family variant format

### CODE REVIEW

File: import_and_export_data/formats/product.rst [hunk 1]
```diff
@@ -26,6 +27,7 @@ Product properties fields:
 - **enabled**: boolean
 - **family**: family code
 - **groups**: list of product group codes
+- **parent**: list of product models codes
```
Comment (pierallard) [inline import_and_export_data/formats/product.rst]: Sure ?

File: import_and_export_data/formats/product_model.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Product model data structure
+============================
+
+Product models are exported in a file with the following fields:
```
Comment (pierallard) [inline import_and_export_data/formats/product_model.rst]: finish with a .

File: import_and_export_data/formats/product_model.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Attributes fields can be one of these:
+
+- **code** (required): the product model code
+- **clothing_size**: size option code for a simple select option
+- **description-<locale_code>-<channel>**: localizable and scopable attribute for channel mobile one row by local_code dans channel
+- **name-<locale_code>**: localizable attribute one row by local_code
+- **price-EUR**: numeric price value for Euro
+- **price-USD**: numeric price value for US Dollar
```
Comment (pierallard) [inline import_and_export_data/formats/product_model.rst]: why not the same description than above ? like `**price-<currency>**: numeric price value for this currency (example: price-EUR`

File: import_and_export_data/formats/family_variant.rst [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+The import/export file follows this data structure:
+
+- **code** (required): the family code
+- **family** : the family parent
+- **attributes**: the list of the family attribute codes separated with a comma
+- **label-<locale_code>**: each label in a dedicated column (See :doc:`localized-labels`)
+- **attribute_as_label**: the attribute whose value will be used as the product label on product datagrid
+- **requirements-<channel_code>**: each requirement in a dedicated column depending on the channel
+- **variant-axes_<1/2>**: the list of the axis variant attributes (5 max)
```
Comment (pierallard) [inline import_and_export_data/formats/family_variant.rst]: I may put |  instead of /

File: import_and_export_data/formats/family_variant.rst [hunk 5]
```diff
@@ -0,0 +1,19 @@
+Family variant data structure
+=============================
+
+The import/export file follows this data structure:
+
+- **code** (required): the family code
+- **family** : the family parent
+- **attributes**: the list of the family attribute codes separated with a comma
```
Comment (Delphineray) [inline import_and_export_data/formats/family_variant.rst]: Not in the family variant format

File: import_and_export_data/formats/family_variant.rst [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+- **label-<locale_code>**: each label in a dedicated column (See :doc:`localized-labels`)
+- **attribute_as_label**: the attribute whose value will be used as the product label on product datagrid
```
Comment (Delphineray) [inline import_and_export_data/formats/family_variant.rst]: Not in the family variant format

File: import_and_export_data/formats/family_variant.rst [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+- **requirements-<channel_code>**: each requirement in a dedicated column depending on the channel
```
Comment (Delphineray) [inline import_and_export_data/formats/family_variant.rst]: Not in the family variant format

File: import_and_export_data/formats/product.rst [hunk 8]
```diff
@@ -18,6 +18,7 @@ Attributes fields can be one of these:
 Associations fields:
 
 - **<association-type-code>-groups**: groups list for the <association-type-code> association type
+- **<association-type-code>-product_models**: list of product models identifiers in this association type
```
Comment (Delphineray) [inline import_and_export_data/formats/product.rst]: "codes" is better than "identifiers"

File: import_and_export_data/formats/product.rst (hunk 1 above)
Comment (Delphineray) [inline import_and_export_data/formats/product.rst]: product model code (for a variant product)

File: import_and_export_data/formats/product_model.rst (hunk 2 above)
Comment (Doodoune) [reply import_and_export_data/formats/product_model.rst]: All the format have the same sentence, finished with ':' so I prefer standardization

File: import_and_export_data/formats/family_variant.rst [hunk 9]
```diff
@@ -0,0 +1,16 @@
+Family variant data structure
+=============================
+
+The import/export file follows this data structure:
+
+- **code** (required): the family code
+- **family** : the family parent
```
Comment (SamirBoulil) [inline import_and_export_data/formats/family_variant.rst]: `The parent family` or even `the family to which this family variant belongs` as you whish

File: import_and_export_data/formats/family_variant.rst [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+- **label-<locale_code>**: each label in a dedicated column (See :doc:`localized-labels`)
+- **variant-axes_<1|2>**: the list of the axis variant attributes (5 max)
+- **variant_attributes_<1|2>**: the list of the variant attributes of the axis
```
Comment (SamirBoulil) [inline import_and_export_data/formats/family_variant.rst]: ...for the axis` (?)

File: import_and_export_data/formats/product_model.rst (hunk 2 above)
Comment (SamirBoulil) [reply import_and_export_data/formats/product_model.rst]: Well it's a documentation, and it should be english ^^. I agree with @pierallard it should a '.' to end the sentence.

File: import_and_export_data/formats/family_variant.rst (hunk 9 above)
Comment (Doodoune) [reply import_and_export_data/formats/family_variant.rst]: Fix

File: import_and_export_data/formats/family_variant.rst (hunk 10 above)
Comment (Doodoune) [reply import_and_export_data/formats/family_variant.rst]: Fix

File: import_and_export_data/formats/product_model.rst (hunk 2 above)
Comment (Doodoune) [reply import_and_export_data/formats/product_model.rst]: Fix for product & product model.

**Pull request merged**